PyPortal/PyPortal_Halloween_Countdown/settings.toml

```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: 2019 Kattni Rembor for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+
+# This file is where you keep private settings, passwords, and tokens!
+# If you put them in the code you risk committing that info or sharing it
+
+CIRCUITPY_WIFI_SSID="your-wifi-ssid"
+CIRCUITPY_WIFI_PASSWORD="your-wifi-password"
+timezone="America/New_York"  # http://worldtimeapi.org/timezones
```

Comment:
I think this may need to have these also like the old secrets.py file did.
```
ADAFRUIT_AIO_USERNAME="my_username"
ADAFRUIT_AIO_KEY="my_key"
```
I believe they might get used when the code checks the time via adafruit io.

Reply:
latest commit adds these placeholders to settings.toml example file.

Reply:
@FoamyGuy I see that you merged it, should I add these in the next PR?

Reply:
I added them with a commit before merging. So there shouldn't be anything needed for this part in the next PR.

Reply:
Ahh didn't catch that